lib/PAUSE.pm

```diff
@@ -100,6 +100,7 @@ $PAUSE::Config ||=
      HTTP_ERRORLOG => '/usr/local/apache/logs/error_log', # harmless use in cron-daily
      INCOMING => $IS_PAUSE_US ? 'ftp://localhost/incoming/' : 'ftp://pause.perl.org/incoming/',
      INCOMING_LOC => '/home/ftp/incoming/',
+     INCOMING_TMP => '/tmp/',
```

Comment:
Why are we putting MFA-enabled-user uploads in a different place?

Reply:
Isn't it confusing putting distributions that are not yet confirmed in the same /incoming/ directory? If not, I'm happy to put them under /incoming/ and remove some code.

Reply:
I hadn't gotten far enough, when reading this (commit by commit) to understand that the process was "upload, then confirm".  Having a second location makes sense.  (Obviously, it'd be great to not have to confirm after upload.)

Reply:
> Obviously, it'd be great to not have to confirm after upload

If you pass a valid otp field value when you upload a file, the otp validation is done at the time of upload. However, the valid otp may change while uploading a large file. So we need to give users a chance to try a new otp without uploading the same file again.